docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -318,6 +318,39 @@ sometimes resolved by additionally installing the ``apptainer-setuid`` package.
       ``slurm.slots_per_node`` to enable multiple CPUs to be used on each node. Without
       ``slurm.slots_per_node`` the job will request ``slots_per_trial`` nodes.
 
+******************
+ PBS Known Issues
+******************
+
+-  With PBS Workload Manager, when users try to view a Resource Pool Configuration, the Accelerator
```

Comment:
Is this the right voice for user documentation?

Reply:
Updated the language and added screenshots in the PR description.